gradient/api_sdk/archivers.py

```diff
@@ -0,0 +1,120 @@
+import fnmatch
+import os
+import zipfile
+
+import progressbar
+
+from gradient.api_sdk.logger import MuteLogger
```

Comment:
Please, try to avoid using full path (at least in SDK) - it will make in easier to separate the SDK part from the CLI.

Use `from .logger import ..` or similar